artemis/src/main/java/tech/pegasys/artemis/cli/BeaconNodeCommand.java

```diff
@@ -217,12 +219,33 @@ private int handleParseException(final CommandLine.ParameterException ex, final
     errorWriter.println(ex.getMessage());
 
     if (!CommandLine.UnmatchedArgumentException.printSuggestions(ex, outputWriter)) {
+      appendErrorMessageToHelp(ex.getCommandLine(), ex.getMessage());
       ex.getCommandLine().usage(outputWriter, CommandLine.Help.Ansi.AUTO);
     }
 
     return ex.getCommandLine().getCommandSpec().exitCodeOnInvalidInput();
   }
 
+  private void appendErrorMessageToHelp(CommandLine cmd, String message) {
+    final String SECTION_KEY_NEW_HEADING = "errorHeading";
+    final String SECTION_KEY_NEW_DETAILS = "errorContent";
```

Comment:
nit: These are not constants so should use normal camel case (or be moved to actual constants).

Reply:
fixed